app/models/loan.rb

```diff
@@ -1,8 +1,27 @@
 # frozen_string_literal: true
 
 class Loan < ApplicationRecord
-  belongs_to :cart
+  DEFAULT_CHECKOUT_PERIOD_IN_DAYS =  14
+
   belongs_to :carrier
 
-  validates :cart, :carrier, :due_date, presence: true
+  belongs_to :member, class_name: "User"
+  validates :member, presence: true
```

Comment:
Unnecessary validation here, since Rails 5, all `belongs_to` associations are validated on the presence by default, unless you specify `optional: true` on the association